ui/guest/src/classes/ContentController.js

```diff
@@ -864,6 +890,19 @@ function reducer(lookupTable, model) {
 
 }
 
+function getResult(model, fieldId, index) {
```

Comment:
Make sure to test with the different level possibilities. 

- Collections at first level
- Nested collections